integration/simulation/network/basicsocket.go

```diff
@@ -105,7 +105,9 @@ func (n *basicNetworkOfSocketNodes) TearDown() {
 	go func() {
 		for _, m := range n.ethNodes {
 			t := m
-			go t.Stop()
+			go func() {
+				_ = t.Stop()
```

Comment:
@otherview Why bother doing this, if we're going to ignore the error anyway? Maybe just suppress the linter?

Reply:
Good point 👍